### ISSUE TITLE
Hugsql 0.5.3 still requires SQL files on uberjar compilation (with leiningen)

### ISSUE THREAD

**Issue by jngbng** [layerware/hugsql#138]
#126 explains that SQL files can be stripped from JAR. (when it builds with depstar.)
But it does not work with leiningen. Uberjar with option 'aot all & omit-source' still throws 'Can not read file' exception.
Test project: https://github.com/jngbng/hugsql_aot

**Comment by csummers** [comment]
Yep, this is still broken.  And, I can't get this to work with depstar either, so it's not just a leiningen issue.  I'll have to switch back to macros.  My apologies!
